.github/CODEOWNERS

```diff
@@ -23,145 +23,184 @@
 ##################
 /eng/                                  @Azure/azure-mcp @Azure/azure-sdk-eng
 /.github/                              @Azure/azure-mcp @Azure/azure-sdk-eng
-/.config/                              @Azure/azure-mcp @Azure/azure-sdk-eng
 
 ##################
 # Services
 ##################
 # PRLabel: %area-AKS
-/src/Areas/Aks/                        @jongio @Azure/azure-mcp
-/tests/Areas/Aks/                      @jongio @Azure/azure-mcp
-/infra/services/aks.bicep              @jongio @Azure/azure-mcp
+/areas/aks/src/                        @jongio @Azure/azure-mcp
+/areas/aks/tests/                      @jongio @Azure/azure-mcp
 
 # ServiceLabel: %area-AKS
 # ServiceOwners:                       @jongio
 
 
 # PRLabel: %area-AppConfig
-/src/Areas/AppConfig/                  @JonathanCrd @conniey @Azure/azure-mcp
+/areas/appconfig/src/                  @JonathanCrd @conniey @Azure/azure-mcp
+/areas/appconfig/tests/                @JonathanCrd @conniey @Azure/azure-mcp
```

Comment:
Any reason these aren't just pathed to the root areas folder?

`/areas/appconfig/`